frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -48,11 +48,8 @@ page-section.c-section(:title='L("Invite links")')
           )
             i.icon-ellipsis-v
         td.c-state
-          i18n.c-state-description {{ item.state.description }}
-          i18n.c-state-expire(
-            v-if='item.state.expireInfo'
-            :class='item.state.isExpired ? "expired" : ""'
-          ) {{ item.state.expireInfo }}
+          i18n.c-state-description {{ item.status }}
```

Comment:
For now this PR seems good, but later we'll need to also add the "Anyone" link. The logic for it is being implemented at #747. Once it's merged you can create another PR and address that.

Reply:
Ok. if #747 is merged earlier than this PR, I will implement it and let you know for a review!

Reply:
Another thing I noticed: Here you are using the status directly as a copy. We can't do that because of translations.  So you need to map each status to its correct translation. You can use the object trick again, but this time, each key is a string instead of an object. You can see an example at `ProposalTemplate.vue`, method `submitTextNonProposal()`

Reply:
Thanks! :) I just checked your code and am liking how you use objects for switching among multiple options. I will push a commit maybe sometime later today. 👍

Reply:
@sandrina-p Just pushed a commit for it!